dispatcher/main.go

```diff
@@ -201,6 +201,27 @@ func getVolumes(run *models.Run) (volumes []corev1.Volume) {
 			},
 		})
 
+	if len(droidMetadata.SecretFiles) == 0 {
+		return
+	}
```

Comment:
It should be unnecessary to check the size of the secret files. If the collection is empty, the following loop will do nothing.

Reply:
Fixed